boot/zephyr/Kconfig

```diff
@@ -1096,6 +1096,14 @@ config MCUBOOT_ACTION_HOOKS
 	  'mcuboot_status_type_t' is listed in
 	  boot/bootutil/include/bootutil/mcuboot_status.h
 
+config FIND_NEXT_SLOT_HOOKS
+	bool "Enable hooks for finding the next active slot"
+	help
+	  This will call a handler when the MCUboot searches for the best slot
+	  for an image. By default, a slot with the highest version is selected.
```

Comment:
No ',' after by default. This is not weak interruption, you can not remove "By default", without changing the meaning of the entire sentence.